src/commands/publish.ts

```diff
@@ -72,6 +72,17 @@ export const builder: CommandBuilder = (yargs: Argv) => {
         'Source revision (git SHA or tag) to publish (if not release branch head)',
       type: 'string',
     })
+    .option('merge-target', {
+      alias: 'm',
+      description:
+        'Target branch to merge into. Uses the default branch from GitHub as a fallback',
+      type: 'string',
+    })
+    .option('remote', {
+      default: 'origin',
+      description: 'The git remote to use when pushing',
+      type: 'string',
+    })
```

Comment:
If I'm reading things right there is already an `argv.remote` on master today. Am I reading things right? Where does it come from? Or is that prepare vs. publish ... 🤔

Reply:
>Or is that prepare vs. publish ... 🤔

This. We had this on `prepare` but not on `publish` since we were interacting with GitHub directly, not needing to know the remote for the local `git`. Now tables have turned.